.gitignore

```diff
@@ -62,7 +51,7 @@ jspm_packages
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon


```

Comment:
What is this? It looks like some misinterpreted escape characters from bash maybe? I might be phrasing that wrong but I think you know what I mean. Either way, correct it.